### PR TITLE
Use old percentage syntax for log messages

### PR DESCRIPTION
**Pull request by johannaengland**

### CODE REVIEW

**Review by lunkwill42 — COMMENTED** [review]
I would want this :)

File: pyproject.toml [hunk 1]
```diff
@@ -40,4 +40,4 @@ line-length = 120
 output-format = "full"
 
 [tool.ruff.lint]
-select = ["E9", "F63", "F7", "F82"]
+select = ["E9", "F5", "F63", "F7", "F82"]
```
Comment (hmpf) [inline pyproject.toml:43]: Yes

File: src/argus_htmx/incidents/views.py [hunk 2]
```diff
@@ -123,7 +123,7 @@ def incident_detail_close(request, pk: int):
             request.user,
             description=form.cleaned_data.get("description", ""),
         )
-        LOG.info(f"{{ incident }} manually closed by {{ request.user }}")
+        LOG.info(f"{ incident } manually closed by { request.user }")
```
Comment (hmpf) [inline src/argus_htmx/incidents/views.py:126]: This should use the old percent syntax: `Log.info("str with %", arg1, arg2)`

Comment (hmpf) [reply src/argus_htmx/incidents/views.py:126]: Same with the other log-message.

**Comment by johannaengland** [comment]
Merging since this was discussed during a live code review.

**Pull request merged**